### ISSUE TITLE
Footer tweaks

### ISSUE THREAD

**Issue by jennpb** [wellcometrust/wellcomecollection.org#193]
After discussion with y'all, we decided to:

- Move 'Finding us' and 'Opening times' closer to the elements they describe
- @Norvard to look at hover states and suggest options
- Minimise the number of social accounts to those in Wellcome Collection, which are:
-- Twitter https://twitter.com/explorewellcome
-- Facebook https://www.facebook.com/wellcomecollection/
-- Instagram https://www.instagram.com/wellcomecollection/
-- Soundcloud https://soundcloud.com/wellcomecollection
-- Tumblr http://wellcomecollection.tumblr.com/
-- YouTube https://www.youtube.com/user/WellcomeCollection
- Change text labels on social icons to site name (e.g. Twitter, Soundcloud, etc)
- Opening hours - needs attention, when javascript is disabled. At present can't see opening times and alignment is off. See below:

![screen shot 2017-01-09 at 10 37 56](https://cloud.githubusercontent.com/assets/6051896/21763912/4722a070-d658-11e6-9ed9-7a60de4bcb30.png)

**Comment by davidpmccormick** [comment]
I think the 'tweaks' in this issue have been covered off. The plethoric social icons/labelling thereof probably deserves its own ticket, with this one being good to close once #267 is merged; @jennpb thoughts?

**Comment by jennpb** [comment]
Yup, we can close this one - also duplicated in #213
